sdc/tests/test_series.py

```diff
@@ -2977,6 +3011,51 @@ def test_impl(S):
         S = pd.Series([np.nan, 2., 3., np.inf])
         pd.testing.assert_series_equal(hpat_func(S), test_impl(S))
 
+    def test_series_isna_index(self):
+        def test_impl(S):
+            return S.isna()
+        hpat_func = self.jit(test_impl)
+
+        # column with NA
+        S = pd.Series(data=[np.nan, 2., 3., np.inf], index=[3, 2, 1, 0])
+        pd.testing.assert_series_equal(hpat_func(S), test_impl(S))
+
+    def test_series_isna_named(self):
```

Comment:
You could combine the test with `test_series_isna_str_named` through the loop.

Reply:
done